statsmodels/datasets/utils.py

```diff
@@ -114,28 +114,16 @@ def _get_cache(cache):
 
 
 def _cache_it(data, cache_path):
-    if PY3:
-        # for some reason encode("zip") won't work for me in Python 3?
-        import zlib
-        # use protocol 2 so can open with python 2.x if cached in 3.x
-        data = data.decode('utf-8')
-        open(cache_path, "wb").write(zlib.compress(cPickle.dumps(data,
-                                                                 protocol=2)))
-    else:
-        open(cache_path, "wb").write(cPickle.dumps(data).encode("zip"))
+    import zlib
+    data = data.decode('utf-8')
+    open(cache_path, "wb").write(zlib.compress(cPickle.dumps(data)))
```

Comment:
should this be a `with open`?  don't want to rely on gc to close the file.  fine to leave for follow up

Reply:
Probably, was doing a mechanical for this pass